specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+              }
+            }
+          },
+          "description": "A list of Azure Resource IDs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for an API service call that lists the resource IDs of resources associated with another resource."
+    },
+    "Etag": {
+      "readOnly": true,
+      "type": "string",
+      "description": "A unique read-only string that changes whenever the resource is updated."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Creating",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Updating",
+        "Deleting"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      },
+      "description": "The current state of the resource's creation, deletion, or modification.",
+      "readOnly": true
+    },
+    "AuthorizedGroundstation": {
+      "type": "object",
+      "required": [
+        "groundStation",
+        "expirationDate"
+      ],
+      "readOnly": true,
+      "properties": {
+        "groundStation": {
+          "type": "string",
+          "description": "Groundstation name."
+        },
+        "expirationDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Date of authorization expiration."
+        }
+      },
+      "description": "Authorized groundstation."
+    },
+    "OperationResult": {
+      "description": "Operation Result Entity.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "description": "ID of the resource.",
+          "type": "string"
+        },
+        "name": {
+          "readOnly": true,
+          "description": "Name of the resource.",
+          "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "startTime": {
+          "description": "The operation start time (ISO 8601 UTC standard).",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "endTime": {
+          "description": "The operation end time (ISO 8601 UTC standard).",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "percentComplete": {
+          "type": "number",
+          "format": "double",
+          "minimum": 0,
+          "maximum": 100,
+          "description": "Percentage completed.",
+          "readOnly": true
+        },
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "object",
+            "properties": {}
+          },
+          "x-ms-identifiers": [],
+          "description": "A list of Azure Operations."
```

Comment:
This isn't necessarily a list of operations. I'd just say, "A list of results when the operation returns multiple results."